packages/esm-patient-orders-app/src/lab-results/lab-results-form.test.tsx

```diff
@@ -99,7 +99,7 @@ describe('LabResultsForm', () => {
     const user = userEvent.setup();
     render(<LabResultsForm {...testProps} />);
 
-    const input = await screen.findByLabelText(`Test Concept (0 - 100 mg/dL)`);
+    const input = await screen.findByLabelText(`Test Concept (0 - 100) mg/dL`);
```

Comment:
See I actually think the original text here is better than this version, since the numbers _should_ be grouped with the units.